code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -218,6 +222,7 @@
 	flags_cover = MASKCOVERSEYES
 	resistance_flags = FLAMMABLE
 	flags_inv = HIDEFACE|HIDEFACIALHAIR|HIDESNOUT
+	mutant_variants = NONE
```

Comment:
Put this in the non_anthro_clothing.dm file

Reply:
Can I edit that here or do I have to make a new pr for it?

Reply:
> 

You can do it in this PR.